### PR TITLE
More up to date fixes, fix some dialogue, new areas

### PR DESCRIPTION
**Pull request by Scorchys12**

### CODE REVIEW

File: Module/dlg/daninfo.dlg.json [hunk 1]
```diff
@@ -0,0 +1,373 @@
...
+          "value": ""
+        },
+        "Speaker": {
+          "type": "cexostring",
+          "value": ""
+        },
+        "Text": {
+          "type": "cexolocstring",
+          "value": {
+            "0": "If you're looking for some fun there is a spa out in the Colony that you can relax. A Cantina if you're feeling like speading some creds at the Colony Center. Hmm..Oh! There's a new hotel owned by a wookie, too, if you wish to rest at. If you also feel like this planet is for you, there is an apartment center aswell."
```
Comment (zunath) [inline Module/dlg/daninfo.dlg.json:155]: More grammar stuff here I missed the first time around. Would recommend:

"If you're looking for some fun, you can relax at a spa in the Colony. The cantina is also a fine place if you're up for spending some credits at the center of the Colony. Hmm... Oh! There is a new hotel owned by a Wookiee, too, if you need some rest. Finally, we also offer some apartments if you'd like to extend your stay here as well."

File: Module/dlg/daninfo.dlg.json [hunk 2]
```diff
@@ -0,0 +1,373 @@
...
+          "value": ""
+        },
+        "Speaker": {
+          "type": "cexostring",
+          "value": ""
+        },
+        "Text": {
+          "type": "cexolocstring",
+          "value": {
+            "0": "The wildlife huh? Word has it there some's mutated bol in some crystal caverns. The Kinraths have also gotten a bit feisty with people getting close to their nest near the lake. Becareful when going to the farmlands further north as there is some angry Thune rampaging about."
```
Comment (zunath) [inline Module/dlg/daninfo.dlg.json:204]: Grammar recommendation:

"The wildlife, huh? Word has it there are some mutated bol in the crystal caverns. People have been getting too close to the Kinrath nests near the lake so they've gotten pretty feisty. Be careful when going to the farmlands farther north as an angry Thune was reportedly rampaging about."

File: Module/dlg/daninfo.dlg.json [hunk 3]
```diff
@@ -0,0 +1,373 @@
...
+          "value": ""
+        },
+        "Sound": {
+          "type": "resref",
+          "value": ""
+        },
+        "Text": {
+          "type": "cexolocstring",
+          "value": {
+            "0": "I would like to know the local wildlife."
```
Comment (zunath) [inline Module/dlg/daninfo.dlg.json:347]: Grammar:

"I would like to learn more about the local wildlife."

File: Module/dlg/daninfo.dlg.json (hunk 1 above)
Comment (Scorchys12) [reply Module/dlg/daninfo.dlg.json]: Fixed here through edit here, i think it works?

File: Module/dlg/daninfo.dlg.json (hunk 3 above)
Comment (Scorchys12) [reply Module/dlg/daninfo.dlg.json]: Fixed here too in new commit

**Review by zunath — APPROVED** [review]

**Pull request merged**